bin/run.js

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env node
+/**
+ * This script is a command line runner for JS-Interpreter.
+ * It exposes a few "native" apis that can be used in the
+ * script being interpreted:
+ *
+ * 1. console.log(...args: string[]) - just a pass through to
+ *        node's native console.log implementation
+ *
+ * 2. vm.runInContext(source: string, context: Object) - executes
+ *        the interpreter against the given source, using the context
+ *        object for the global scope.
+ *
+ * Usage:
+ *   js-interpreter <filepath.js>
+ */
+
+window = typeof window === 'undefined' ? global : window;
+var fs = require('fs');
+const yargs = require('yargs');
```

Comment:
I think I was just flip flopping between which version of node I wanted to support. Leaning towards just supporting the later versions of node, which all support const. So I should probably just get rid of all the var declarations.